### PR TITLE
Add RustBridge workshop

### PR DESCRIPTION
**Pull request by gandro**
This adds RustBridge as the first workshop. Likely to conflict with #30 due to edits in the speakers collection.

image

@oe Sorry, I forgot to ask earlier: Do you have a portrait which you would like to add? Also, do you want your Twitter/GitHub handle shown?

### CODE REVIEW

**Comment by apiraino** [comment]
@gandro I will solve the conflicts, no big deal.

Would it make sense to push in this PR the other workshops, too?

**Comment by gandro** [comment]
Yeah, I'll try to add the others soon! Might end up being a second PR.

**Comment by apiraino** [comment]
great! In case let me know, I can add a couple of commits

**Comment by oe** [comment]
hm, can we add a picture to my profile?

File: _data/speakers.yml [hunk 1]
```diff
@@ -44,3 +44,14 @@ lee:
       link: https://twitter.com/_lbaillie
   desc: |
     Lee Baillie herds Rubies and wrangles JavaScripts (occasionally also tending to the Rust garden) at Tilde in Portland, OR, mainly in the context of building Skylight, a Rails performance profiler. In their spare time, they enjoy hanging out with their dog JPEG, exploring the beautiful outdoors of the PNW, and lifting heavy things only to put them down again.
+
+# Workshops
+
+olivia:
+  name: Olivia Hugger
+  thumbnailUrl: ~
+  links:
```
Comment (ghost) [inline _data/speakers.yml:53]: My Twitter handle would be https://twitter.com/dog_awoo

File: _data/speakers.yml [hunk 2]
```diff
@@ -44,3 +44,14 @@ lee:
       link: https://twitter.com/_lbaillie
   desc: |
     Lee Baillie herds Rubies and wrangles JavaScripts (occasionally also tending to the Rust garden) at Tilde in Portland, OR, mainly in the context of building Skylight, a Rails performance profiler. In their spare time, they enjoy hanging out with their dog JPEG, exploring the beautiful outdoors of the PNW, and lifting heavy things only to put them down again.
+
+# Workshops
+
+olivia:
+  name: Olivia Hugger
+  thumbnailUrl: ~
```
Comment (ghost) [inline _data/speakers.yml]: Picture-wise, you can go with this one 😁 

image

**Comment by gandro** [comment]
Great portrait :smile: 

Pushed changes and added social card. I will publish this Tuesday noon together with the blog post.

**Pull request merged**